### ISSUE TITLE
Improved Saved Settings

### ISSUE THREAD

**Issue by acidreian666** [Exslims/MercuryTrade#269]
At-least once every 2-3 days Mercury Stops notifying incoming/outgoing trades. Mercury is manually Closed and restarted. This has happening to me more then 10 times.
But afterwards all setting are lost due to the way settings are not stored in a file but in some temp memory?

Please configure for either a backup or a means to save settings to file in such this happens again taking hours to reconfigure for heavy trading is a pain in the ass. 
Or add the feature to stop Mercury from having some sort of permission to overwrite setting where ever you stored them.,

**Comment by LordBlick** [comment]
Yes its very frustrating to lost settings without ability to backup them, but in this case it is a false guess.
Although in [Troubleshooting section](https://github.com/Exslims/MercuryTrade/wiki/Troubleshooting) you may find a tip about where user config is stored. Search for `configuration.json` file.
